oblib/taxonomy.py

```diff
@@ -235,25 +749,13 @@ class Taxonomy(object):
     def __init__(self):
         """Taxonomy constructor."""
 
-        # Temporary fix for the circular dependency issue
-        from oblib import taxonomy_semantic, taxonomy_units
-
-        self.semantic = taxonomy_semantic.TaxonomySemantic()
-        self.types = taxonomy_types.TaxonomyTypes()
-        self.units = taxonomy_units.TaxonomyUnits()
-        self.numeric_types = taxonomy_misc.TaxonomyNumericTypes()
-        self.generic_roles = taxonomy_misc.TaxonomyGenericRoles()
-        self.ref_parts = taxonomy_misc.TaxonomyRefParts()
-        self.documentation = taxonomy_misc.TaxonomyDocumentation()
-
-
-# Accessor for singleton Taxonomy object:
-m_singletonTaxonomy = None
-
+        from oblib import taxonomy_loader
```

Comment:
I don't see a reason not to put this import at the head of the file.

Reply:
This was a temporary fix for the circular dependency issue between 2 taxonomy modules. Since the modules have been combined, this import can be moved to the top of the file.

Reply:
I could not get this to work correctly with Python 3.4 with it at the top of the file although it works fine with 3.5 and 3.6.  I am leaving it here for now and we can move it on 1.1.0 when we move the Python version forward.  I'll create an issue.